src/model.jl

```diff
@@ -82,7 +89,8 @@ function (u::Unet)(x::AbstractArray)
   up_x2 = u.up_blocks[2](up_x1, x2)
   up_x3 = u.up_blocks[3](up_x2, x1)
   up_x5 = u.up_blocks[4](up_x3, op)
-  tanh.(u.up_blocks[end](up_x5))
+  u.up_blocks[end](up_x5)
```

Comment:
As far as I know there is no tanh activation function in the last UNET layer only softmax (see sect 3 of the paper). The, Flux recommends using logitcrossentropy for numerical stability instead of cross entropy and softmax separated hence I removed softmax altogether as it is included in the loss function itself.

Reply:
In that case, I think it would be nice if we could have the default be identity and have a parameter in Unet that can control the activation